### DISCUSSION TITLE
Allow "visible" incoming connections on installed RustDesk

### DISCUSSION THREAD

**Discussion by danielmarschall** [rustdesk/rustdesk#6590] [Feature Request] [answered]
I noticed the following behavior:

- When RustDesk is not installed, then the remote person can give me their ID and password, and I can access their current Deskop.

- When RustDesk is installed, I can access the system without the user noticing (except if I login using their user account). However, I cannot enter their existing Deskop. When the remote person wants that I help them, I can't show them anything or see what they are doing, because I end up in the Windows login screen.

Can I somehow choose, if I want to connect to the Login screen (i.e. for doing something in the background without interrupting the user), or in the existing Windows session, to see what the user is doing?

Example:
At the right side we see the partner computer which has RustDesk open. Imagine they tell me their ID.
On the left side is what I see when I try to connect.

![image](https://github.com/rustdesk/rustdesk/assets/28412477/125c56cb-bbee-4bc1-a4f2-08f0ac32bea1)

**Comment by subscribernamegoeshere** [comment]
you are mixing rustdesk usage into an rdp session. i have not yet understood what exactly in this rdp scenario is currently supported and what not. there are bugreports and feature requests regarding rustdesking into/with RDP windows sessions.

**Reply by danielmarschall** [reply]
I think one solution could be that the ID number is not dependent on the machine, but on the machine+user.
If the ID would be independent of machine+user, then there would be an ID for connecting to the RDP login screen (System account?) and a different ID for connecting "on-demand", i.e. the user is connected in a RDP session and needs help.

Though I am afraid this change would not be possible because the impact would be too big (since a lot of users remember their existing ID).

**Reply by danielmarschall** [reply]
Actually, it is much easier.

When I have a fixed password, then I can connect to the login screen.

When I enter the password that the user sees on their RustDesk window, then I should be connected to the session, instead of the Windows login screen. This would solve the problem. However, it would be important that different users on the system have different RustDesk one-time-passwords.

**Comment by danielmarschall** [comment, marked as answer]
#323
